rdt-open-reader/src/main/java/io/ona/rdt/interactor/LoginInteractor.java

```diff
@@ -30,9 +30,7 @@ public void scheduleJobsImmediately() {
 
     @Override
     public void login(WeakReference<BaseLoginContract.View> view, String userName, char[] password) {
-        if (NetworkUtils.isNetworkAvailable()) {
-            getSharedPreferences().saveForceRemoteLogin(true, userName);
-        }
+        getSharedPreferences().saveForceRemoteLogin(NetworkUtils.isNetworkAvailable(), userName);
```

Comment:
Other than adding brevity, does this fix any underlying issue? The code seems to be doing the same thing as before.

Reply:
Isn't `remoteLogin` sharedpreference reset to `false` after a successful remote login?

Reply:
yes, it has set false here https://github.com/opensrp/opensrp-client-core/blob/690c2bde513821c0a377be5e0530e906834cbfb3/opensrp-app/src/main/java/org/smartregister/service/UserService.java#L410 previously I have faced an issue with local login when I turned my network off but it's not reproducing yet although I tried many times. For the safe side, I have used this logic.

Reply:
I don't think there's much difference in the logic. Although the first time you try logging in on a fresh app, the app should attempt remote login even if the network is not available. I'd keep this logic as is unless it clearly fixes a bug.

Reply:
I think the bug will reproduce. You can see here https://github.com/opensrp/opensrp-client-core/blob/690c2bde513821c0a377be5e0530e906834cbfb3/opensrp-app/src/main/java/org/smartregister/util/SyncUtils.java#L65 when the session expired then force_remote set true https://github.com/opensrp/opensrp-client-core/blob/690c2bde513821c0a377be5e0530e906834cbfb3/opensrp-app/src/main/java/org/smartregister/service/UserService.java#L413

Reply:
I don't follow. What is wrong with force remote login being set to true on logout?

Reply:
Force remote login should set to be `false` on any case of logout.

Step to reproduce a bug:
1. Enable network and do login.
2. After login left the app in the background.
3. Resume the app after 2 minutes i.e assuming the session has been expired.
4. The app will logout and take you to the login screen with a session expiry message.
5. Now just disabled the internet and try to login again.
6. Now login will be failed every time until you turn the network on.
7. That's a bug because `force_remote login` is not false currently.

Reply:
I think this is the correct behaviour. On forced logout, you should not be able to login without internet because you need to authenticate with the server.

Reply:
OK. Closing the PR because there is no longer need to open this PR.